BunqSdk/Security/SecurityUtils.cs

```diff
@@ -18,76 +18,76 @@ public class SecurityUtils
         /// <summary>
         /// Error constants.
         /// </summary>
-        private const string ERROR_COULD_NOT_VERIFY_RESPONSE = "Could not verify server response.";
+        private const string ErrorCouldNotVerifyResponse = "Could not verify server response.";
 
         /// <summary>
         /// Constants for formatting the request textual representation for signing.
         /// </summary>
-        private const string NEWLINE = "\n";
-        private const string FORMAT_METHOD_AND_ENDPOINT_STRING = "{0} /v1/{1}";
-        private const string HEADER_NAME_PREFIX_X_BUNQ = "X-Bunq-";
-        private const string DELIMITER_HEADER_VALUE = ",";
-        private const string FORMAT_HEADER_STRING = "{0}: {1}";
+        private const string Newline = "\n";
```

Comment:
@OGKevin do you want these constants grouped as well? Could use Format prefix.

Reply:
Ah yeas `Format` prefix would indeed be great 👌.